boards.txt

```diff
@@ -15,222 +15,77 @@
-adafruit_trinket_m0.upload.use_1200bps_touch=true
-adafruit_trinket_m0.upload.wait_for_upload_port=true
-adafruit_trinket_m0.upload.native_usb=true
-adafruit_trinket_m0.build.mcu=cortex-m0plus
-adafruit_trinket_m0.build.f_cpu=48000000L
-adafruit_trinket_m0.build.usb_product="Trinket M0"
-adafruit_trinket_m0.build.usb_manufacturer="Adafruit"
-adafruit_trinket_m0.build.board=TRINKET_M0
-adafruit_trinket_m0.build.core=arduino
-adafruit_trinket_m0.build.extra_flags=-DCRYSTALLESS -DADAFRUIT_TRINKET_M0 -D__SAMD21E18A__ {build.usb_flags}
-adafruit_trinket_m0.build.ldscript=linker_scripts/gcc/flash_with_bootloader.ld
-adafruit_trinket_m0.build.openocdscript=openocd_scripts/arduino_zero.cfg
-adafruit_trinket_m0.build.variant=trinket_m0
-adafruit_trinket_m0.build.variant_system_lib=
-adafruit_trinket_m0.build.vid=0x239A
-adafruit_trinket_m0.build.pid=0x801E
-adafruit_trinket_m0.bootloader.tool=openocd
-adafruit_trinket_m0.bootloader.file=trinket/bootloader_uf2.bin
-
-adafruit_itsybitsy_m0.name=Adafruit ItsyBitsy M0
-adafruit_itsybitsy_m0.vid.0=0x239A
-adafruit_itsybitsy_m0.pid.0=0x800F
-adafruit_itsybitsy_m0.vid.1=0x239A
-adafruit_itsybitsy_m0.pid.1=0x000F
-adafruit_itsybitsy_m0.vid.2=0x239A
-adafruit_itsybitsy_m0.pid.2=0x8012
-adafruit_itsybitsy_m0.upload.tool=bossac
-adafruit_itsybitsy_m0.upload.protocol=sam-ba
-adafruit_itsybitsy_m0.upload.maximum_size=262144
-adafruit_itsybitsy_m0.upload.use_1200bps_touch=true
-adafruit_itsybitsy_m0.upload.wait_for_upload_port=true
-adafruit_itsybitsy_m0.upload.native_usb=true
-adafruit_itsybitsy_m0.build.mcu=cortex-m0plus
-adafruit_itsybitsy_m0.build.f_cpu=48000000L
-adafruit_itsybitsy_m0.build.usb_product="ItsyBitsy M0 Express"
-adafruit_itsybitsy_m0.build.usb_manufacturer="Adafruit"
-adafruit_itsybitsy_m0.build.board=ITSYBITSY_M0
-adafruit_itsybitsy_m0.build.core=arduino
-adafruit_itsybitsy_m0.build.extra_flags=-DCRYSTALLESS -DADAFRUIT_ITSYBITSY_M0 -D__SAMD21G18A__ {build.usb_flags}
-adafruit_itsybitsy_m0.build.ldscript=linker_scripts/gcc/flash_with_bootloader.ld
-adafruit_itsybitsy_m0.build.openocdscript=openocd_scripts/arduino_zero.cfg
-adafruit_itsybitsy_m0.build.variant=itsybitsy_m0
-adafruit_itsybitsy_m0.build.variant_system_lib=
-adafruit_itsybitsy_m0.build.vid=0x239A
-adafruit_itsybitsy_m0.build.pid=0x800F
-adafruit_itsybitsy_m0.bootloader.tool=openocd
-adafruit_itsybitsy_m0.bootloader.file=itsybitsy/samd21_sam_ba.bin
+mcci_catena_4460.name=Catena4460
+mcci_catena_4460.vid.0=0x040E
+mcci_catena_4460.pid.0=0x001A
+mcci_catena_4460.upload.tool=bossac
+mcci_catena_4460.upload.protocol=sam-ba
+mcci_catena_4460.upload.maximum_size=262144
+mcci_catena_4460.upload.use_1200bps_touch=true
+mcci_catena_4460.upload.wait_for_upload_port=true
+mcci_catena_4460.upload.native_usb=true
+mcci_catena_4460.build.mcu=cortex-m0plus
+mcci_catena_4460.build.f_cpu=48000000L
+mcci_catena_4460.build.usb_product="Catena4460"
+mcci_catena_4460.build.usb_manufacturer="MCCI"
+mcci_catena_4460.build.board=SAMD_FEATHER_M0_EXPRESS
```

Comment:
Really seems wrong. We shoudn't be using SAMD_FEATHER_M0_EXPRESS, as this is not an EXPRESS.